download/download.html.haml

```diff
@@ -156,12 +150,6 @@ title: Download
             <version>#{site.pom.latest.version}</version>
           </dependency>
         ----
-    %div(id="execution-server-nonfinal" class="tab-pane fade")
-      :asciidoc
-        * image:download.png[Download] *Download OptaPlanner Execution Server #{site.pom.latest.version}*
-        ** #{site.pom.latest.kieExecutionServerZip}[Any application server]
-        ** link:releaseNotes/releaseNotes#{site.pom.latest.releaseNotesVersion}.html#_execution_server[Release notes #{site.pom.latest.releaseNotesVersion} (OptaPlanner)]
-        ** License: link:../code/license.html[Apache License 2.0] - Date: #{site.pom.latestFinal.releaseDate.strftime('%a %-d %B %Y')}
```

Comment:
There's "execution-server-nightly" on line 176/164 a few lines below. I think that can be removed too.

Reply:
good catch, thank you.